app/ont/ont.rb

```diff
@@ -20,6 +24,13 @@ def self.direct_request_attributes
     request_attributes - associated_request_attributes
   end
 
+  def self.library_factory(request:, library_attributes:)
```

Comment:
Same here I think this should be `raise StandardError, 'Unsupported'` for the time being and then we support it if there becomes a need for it.